bin/generate_trec_covid_round1_OSHU_RUN2.py

```diff
@@ -0,0 +1,215 @@
+
+import sys
+sys.path.insert(0, "./")
+
+import pandas as pd
+import numpy as np
+#import torch
+import os
+from tqdm.auto import tqdm
+import json
+from pyserini.search import pysearch
+import xml.etree.ElementTree as ET
+import requests
+import urllib.request
+from trectools import misc, TrecRun, TrecQrel, procedures
+from pyserini.analysis.pyanalysis import get_lucene_analyzer, Analyzer
+import nltk
+from nltk.corpus import stopwords
+
+#Round 1 indexes
+#Replace with url to folder containing your index
+R1_fulltext = 'lucene-index-covid-full-text-2020-04-10'
+
+#Download round 1 topics and parse into dataframe
+tree = ET.fromstring(requests.get('https://ir.nist.gov/covidSubmit/data/topics-rnd1.xml').text)
+topicid = []
+query = []
+question = []
+narrative = []
+for child in tree.iter():
+    tag =child.tag
+    text = child.text
+    attrib = child.attrib
+    if (tag == 'topic'):
+        topicid.append(attrib['number'])
+    if (tag == 'query'):
+        query.append(text)
+    if (tag == 'question'):
+        question.append(text)
+    if (tag == 'narrative'):
+        narrative.append(text)
+
+#Join to CSV
+my_dict  = {'Topic':topicid, 'Query':query, 'Question':question , 'Narrative':narrative}
+R1_topics = pd.DataFrame(my_dict)
+R1_topics = R1_topics[['Topic', 'Query', 'Question', 'Narrative']]
+
+curr_dir = os.getcwd()
+Pyserini_files = os.path.join(curr_dir, 'Round1_data')
+if (os.path.exists(Pyserini_files) == False):
+    os.mkdir(Pyserini_files)
+
+#Topics
+full_searcher = pysearch.SimpleSearcher(R1_fulltext)
+
+#Configure searcher parameters
+full_searcher.set_bm25_similarity(k1=1.5, b=0.4)
+full_searcher.set_lm_dirichlet_similarity(mu = 2000)
+full_searcher.set_rm3_reranker(fb_terms=10, fb_docs=10, original_query_weight=0.5)
+
+#Stopwords for tokenization - manual review
+stopwords_manual = ['seek', 'seeking', 'look', 'looking', 'studies', 'study', 'information',
+             'about', 'range', 'studies', 'its', 'coronaviru',
+            'other', '2', '19', 'well', ' will', 'from', 'have', 'more', 'covid', 'any', 'what',
+            'should', 'may', 'due', 'help', 'non', 's', 'those', 'people', 'ways', 'all', 'gain',
+            'possible', 'toward', 'specifically', 'learned', 'number', 'proportion', 'including',
+            'etc', 'still', 'while', 'human', 'specific', 'result', 'results', 'assess', 'need',
+            'between', 'take', 'taking', 'patient', 'type', 'cause' ,'frequency', 'less', 'face',
+            'likely', 'infect', 'upon', 'develop', 'represent', 'promising', 'step', 'related',
+            'papers', 'describe', 'also', 'relevant', 'who', 'show', 'science', 'basic', 'complete',
+            'do', 'how', 'been', 'against', 'use', 'to', 'had', 'has', 'approach', 'Studies', 'Stud', 'Inst', 'Divi' ,'Thomae',
+            'Brigham', 'Young', 'Univ', 'studies', 'volition', 'severe acute respiratory syndrome', 'affect', 'affected']
+
+#NLTK stopwords
+
```

Comment:
```suggestion
nltk.download('stopwords')
```
nltk needs to download the stopwords